controllers/userController.js

```diff
@@ -1,55 +1,69 @@
+import jwt from "jsonwebtoken";
 import User from "../models/user.js";
 
 // Get a list of all users
 const getUsers = (req, res) => {
-    
-    // just for testing
-    res.json({ success: "ok" })
+	// just for testing
+	res.json({ success: "ok" });
 };
 
 const getUser = () => {};
```

Comment:
I guess you know that this is empty?

Reply:
yes, i know. it will probably stay empty, because there's no frontend functionality (this would be an admin dashboard, so only nice to have)